src/components/Navigation.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation: React.FC = () => (
+  <nav
+    className="navbar is-light is-fixed-top is-mobile has-shadow"
+    data-cy="Nav"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          className={({ isActive }) => cn('navbar-item', {
+            'is-active': isActive,
+          })}
```

Comment:
It might be worth putting this logic into a separate function to avoid duplication.
```suggestion
        <NavLink
          className={getLinkClass}
```